cfme/tests/cli/test_appliance_console.py

```diff
@@ -1291,7 +1290,14 @@ def test_appliance_console_extend_storage_negative():
             2.
             3. Check there is no extra disk present.
     """
-    pass
+    command_set = ("ap", RETURN, "11")
```

Comment:
@dgaikwad Here it looks like you're running two tests, thoughts are parametrizing the test case against the two different command sets?